### PR TITLE
sf3 support

### PR DESCRIPTION
**Pull request by emgiezet**
Hi all tests are good. Lets go!

### CODE REVIEW

**Comment by saimaz** [comment]
Hi thanks for the PR. With your change we will make conflict for symfony ~2.7. Can you please change `<=3.0` to the `>=2.7` ?

**Comment by emgiezet** [comment]
i will change it to cover range ```>=2.7, <=3.0```

**Comment by emgiezet** [comment]
ok done ;)

File: composer.json [hunk 1]
```diff
@@ -12,8 +12,8 @@
     ],
     "require": {
         "php": ">=5.4",
-        "symfony/serializer": "~2.7",
-        "symfony/options-resolver": "~2.7"
+        "symfony/serializer": ">=2.7,<=3.0",
+        "symfony/options-resolver": ">=2.7,<=3.0"
```
Comment (saimaz) [inline composer.json]: The right way would be: `"symfony/options-resolver": ">=2.7"`

Same for serializer.

I don't see any reason lock to the specific `3.0` version. When Symfony `3.1` will be released we will need another PR to fix this...

Comment (jdreesen) [inline composer.json]: Why not `~2.7|~3.0`? I'm seeing this in many libs with Symfony deps.

Comment (saimaz) [inline composer.json]: @jdreesen, agree, `~2.7|~3.0` makes sense.

**Comment by saimaz** [comment]
@emgiezet can you change that @jdreesen suggested? 

`~2.7|~3.0`

**Comment by emgiezet** [comment]
sure
